slides/00-start/PITCHME.md

```diff
@@ -18,7 +18,7 @@ $(npm bin)/cypress open
 
 +++
 
-![First time you open Cypress](img/cypress-scaffold.png)
+![First time you open Cypress](/img/cypress-scaffold.png)
```

Comment:
should probably move it to subfolder too, no need for top level `/img` - we can keep top level only for images that truly repeat in several presentations